src/views/FindProductInventory.vue

```diff
@@ -328,6 +328,21 @@ export default defineComponent({
       ).then(() => {
         event.target.complete();
       })
+    },
+    async viewProduct(product: any) {
+      product = {
+        productId: product.productId,
+        productName: product.productName,
+        brand: this.getProduct(product.productId).brandName,
```

Comment:
Store this.getProduct(product.productId) in a variable and use

Reply:
Done Sir.